JPF/PubAdvWebScrape.ipynb

```diff
@@ -0,0 +1,605 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2016-11-26T22:41:18.806478",
+     "start_time": "2016-11-26T22:41:12.447122"
+    },
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>ADDR</th>\n",
+       "      <th>LANDLORD</th>\n",
+       "      <th>HPDV</th>\n",
+       "      <th>DOBV</th>\n",
```

Comment:
Ah, got it. It'd be super helpful if you just clarified the last 2 columns. Maybe something like `HPV Violation Count`?